### PR TITLE
Dora artifacts

### PR DESCRIPTION
**Pull request by wuallenwu**

### CODE REVIEW

File: sim/humanoid_gym/envs/dora_config.py [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+            default_joint_angles[joint] = default_positions[joint]
+
+    class control(LeggedRobotCfg.control):
+        # PD Drive parameters:
+        stiffness = Stompy.stiffness()
+        damping = Stompy.damping()
+        # action scale: target angle = actionScale * action + defaultAngle
+        action_scale = 0.25
+        # decimation: Number of control action updates @ sim DT per policy DT
+        decimation = 4  # 100hz
```
Comment (budzianowski) [inline sim/humanoid_gym/envs/dora_config.py:101]: Please change to 10

File: sim/humanoid_gym/envs/dora_env.py [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+            dim=1,
+        )
+
+    def _reward_default_joint_pos(self):
+        """
+        Calculates the reward for keeping joint positions close to default positions, with a focus
+        on penalizing deviation in yaw and roll directions. Excludes yaw and roll from the main penalty.
+        """
+        joint_diff = self.dof_pos - self.default_joint_pd_target
+        left_yaw_roll = joint_diff[:, :2]
```
Comment (budzianowski) [inline sim/humanoid_gym/envs/dora_env.py:378]: Please fix this.

**Review by budzianowski — CHANGES_REQUESTED** [review]

File: sim/humanoid_gym/envs/dora_config.py (hunk 1 above)
Comment (wuallenwu) [reply sim/humanoid_gym/envs/dora_config.py]: fixed

File: sim/humanoid_gym/envs/dora_env.py (hunk 2 above)
Comment (wuallenwu) [reply sim/humanoid_gym/envs/dora_env.py]: fixed

File: sim/humanoid_gym/envs/dora_config.py [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    class safety:
+        # safety factors
+        pos_limit = 1.0
+        vel_limit = 1.0
+        torque_limit = 0.85
+
+    class asset(LeggedRobotCfg.asset):
+        file = str(stompy_urdf_path())
+
+        name = "stompy"
```
Comment (budzianowski) [inline sim/humanoid_gym/envs/dora_config.py:42]: dora

**Review by budzianowski — APPROVED** [review]

**Pull request merged**